src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/merge_xmls.py

```diff
@@ -70,12 +70,12 @@ def aggregate_test_results(aggregated_results: SubElement, xml_reports: list,
         if aggregated_timestamp is None or xml_timestamp < aggregated_timestamp:
             aggregated_timestamp = xml_timestamp
         for xml_device_entry in xml_results:
-            device_name = xml_device_entry.tag
             if merge_device_suffix and "." in xml_device_entry.tag:
                 device_name = xml_device_entry.tag[:xml_device_entry.tag.find("."):]
```

Comment:
looks like this line is useless now , because device_name will be override in 77

Reply:
Look to next line